fastlane/Fastfile

```diff
@@ -26,6 +26,33 @@ lane :capture_screen do
   screengrab
 end
 
+
+desc "Build debug and test APK for screenshots"
+  lane :build_for_screengrab do
+    gradle(
+      task: 'clean'
+    )
+    #build_android_app(
+     # task: 'assemble',
+     # build_type: 'Debug',
+     # flavor: "off",
+     # project_dir: './'
+   # )
+    build_android_app(
+      task: 'assembleconnectedOffScreenshotsAndroidTest',
```

Comment:
```suggestion
      task: 'app:connectedOffPlaystoreDebugAndroidTest',
```

Reply:
@VaiTon suggested this one with debug. I would have put, but I'm crazy
 task: 'app:assembleconnectedOffPlaystoreScreenshotsAndroidTest',

Reply:
Also I disabled 
 val result = "screenshots"
    //val result = "debug"